src/cool_seq_tool/resources/resources.py

```diff
@@ -0,0 +1,69 @@
+"""Provide paths to shared files, and trigger data acquisition if unavailable."""
+from importlib import resources
+from os import environ
+from pathlib import Path
+
+from wags_tails.ncbi_lrg_refseqgene import NcbiLrgRefSeqGeneData
+from wags_tails.ncbi_mane_summary import NcbiManeSummaryData
+
+_configured_transcript_mappings_path = environ.get("TRANSCRIPT_MAPPINGS_PATH")
+if _configured_transcript_mappings_path:
+    TRANSCRIPT_MAPPINGS_PATH = Path(_configured_transcript_mappings_path)
+    if not TRANSCRIPT_MAPPINGS_PATH.exists():
+        msg = f'No transcript mappings file exists at path {_configured_transcript_mappings_path} defined under env var TRANSCRIPT_MAPPINGS_PATH. Either unset to use default file bundled with cool-seq-tool, or ensure that is available at this location. See the "Environment configuration" section under the Usage page within the documentation for more.'
+        raise FileNotFoundError(msg)
+else:
+    TRANSCRIPT_MAPPINGS_PATH: Path = (
+        resources.files(__package__) / "transcript_mapping.tsv"
+    )  # type: ignore[reportAssignmentType]
+
+
+def get_mane_summary() -> Path:
```

Comment:
there's some DRY violation here, but when I wrote up the non-repetitive version it was.... pretty gross so I just left it like this